doc/health-check.md

```diff
@@ -88,6 +89,7 @@ contains: `active` or `passive`.
     The threshold fields of  `unhealthy` are:
     * `active.unhealthy.interval`: Interval between health checks for unhealthy targets (in seconds), the minimum is 1.
     * `active.unhealthy.http_failures`: The number of http failures times to determine the target is unhealthy, the minimum is 1.
+    * `active.req_headers`: Additional request header.
```

Comment:
`Additional request header.` -> `Additional request headers. Array format, so you can fill in multiple headers.`